django/contrib/postgres/fields/hstore.py

```diff
@@ -51,6 +51,20 @@ def formfield(self, **kwargs):
         defaults.update(kwargs)
         return super(HStoreField, self).formfield(**defaults)
 
+    def get_prep_value(self, value):
+        value = super(HStoreField, self).get_prep_value(value)
+
+        if isinstance(value, dict):
+            prep_value = {}
+            for key, val in value.items():
+                if not isinstance(key, six.string_types):
```

Comment:
I wonder if the `isinstance()`condition  results in any performance savings? I tend to think always casting might be simpler.